data/mods/More_Makeshift_Items/mb_recipe_weapons.json

```diff
@@ -0,0 +1,154 @@
+[
+{
+  "type" : "recipe",
+  "result": "makeshift_bayonet",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_MODS",
+  "skill_used": "fabrication",
+  "difficulty": 1,
+  "time": 500,
+  "reversible": true,
+  "autolearn": true,
+  "components": [
+    [
+      [ "spike", 1 ]

```

Comment:
I am using the git shell program running on powershell 2.0 running on a 64bit windows 7 laptop.